WORKSPACE

```diff
@@ -11,8 +11,8 @@ rules_foreign_cc_dependencies()
 load("@rules_python//python:pip.bzl", pip3_install="pip_install")
 
 pip3_install(
-   name = "py_deps",
-   requirements = "//:requirements.txt",
+   name = "tools_pip_deps",
+   requirements = "//tools:requirements.txt",
```

Comment:
i believe the "standard" for pip is to have the requirements in the root.  is that not the case?  and why tools and not bindings/python?

Reply:
Yeah, but in this repo there are multiple `requirements.txt` files in various directories - and the interesting thing is that the requirements in `tools/requirements` seems to be unused, but the pip requirements in the root dir are only applicable for the targets in `tools`.

Reply:
ah you're right. i haven't paid much attention to the python bits. thanks.